src/lib.rs

```diff
@@ -495,12 +517,26 @@ struct ItemsView {
     inner: HashTrieMapSync<Key, PyObject>,
 }
 
+struct ItemViewQuery(Key, PyObject);
```

Comment:
I think you can `#[derive(FromPyObject)]` here.

Reply:
Nice catch! I really appreciate your time and expertise!